### DISCUSSION TITLE
Does the current code supports generation of text + image + image ?

### DISCUSSION THREAD

**Discussion by Wonder1905** [lucidrains/transfusion-pytorch#25] [Q&A]
Hi, does the current code supports generation of text + image + image ? as see in the paper? where they are predicting text image tokens and then again text tokens

**Comment by lucidrains** [comment]
@Wonder1905 it can do anything, any number of modalities, any order

the sky is the limit

**Reply by lucidrains** [reply]
the only thing missing at the moment is allowing for learned encoder / decoder flanking the transformer (unet in paper, but can also be anything)

i'm getting that done this morning, hopefully

**Reply by lucidrains** [reply]
@Wonder1905 once the dust has settled, i'll put together a small demo doing simple arithmetic (1 + 3 = 4), but with numbers substituted with mnist images

**Reply by Wonder1905** [reply]
Niceeeeeeeeeeeeeeee!
